explainer.md

```diff
@@ -0,0 +1,208 @@
+# WebXR Device API - Hand Input
+
+This document describes a design giving developers access to hand-tracking XR systems, building on top of the [WebXR device API](https://immersive-web.github.io/webxr/)
+
+## Use cases and scope
+
+This API primarily exposes the poses of hand skeleton joints. It can be used to render a hand model in VR scenarios, as well perform gesture detection with the hands. It does not provide access to a full hand mesh.
+
+## Accessing this API
+
+This API will only be accessible if a `"hand-tracking"` [XR feature](https://immersive-web.github.io/webxr/#feature-dependencies) is requested.
+
+This API presents itself as an additional field on `XRInputSource`, `hand`. The `hand` attribute will be non-null if the input source supports hand tracking and the feature has been requested.
```

Comment:
Might be worth going into more detail about what we consider to constitute "hand tracking". For example: Will we surface hand poses derived from controller sensors (Knuckles, Touch, etc) here? My assumption thus far has been yes.

However, that kind of derived hand pose is probably generally used a much different way than actual tracked hands (ie: mostly for display purposes), so we'd probably need an API signal indicating the source of the hand pose. Also, I don't see derived hand poses as needing the same level of permission as actual hand tracking (because you could derive it yourself from the gamepad attributes), so the "hand-tracking" feature wouldn't necessarily need to be required in that case and thus we couldn't make the claim that the `hand` attribute will only be non-null if it was requested.

Given that the above likely needs a fair bit more discussion (including wether or not it's in scope at all) I don't consider it blocking for this PR, just want to make sure it's being considered.

Reply:
Yes, but that's https://github.com/immersive-web/webxr-hands-input/issues/7

Reply:
It may still make sense to have the feature request for performance concerns (it's nice to have things be opt in), but have two levels: just simulated poses, or real poses if available. This could be one of those cases where a non-string XR feature is useful (e.g. `{hand_tracking: "simulated"}`)

Reply:
If you chose "simulated" on a device with only a depth sensor to detect hands, would the UA be required to degrade the data in some way to below a level that would require a permissions prompt?

One data point is that the Windows Mixed Reality native platform today has a separate "high-sensitivity" permission for "eye tracking" (explicit user prompt shown), and a "medium-sensitivity" permission for planes/meshes (no initial user prompt, but can revoke access), but we have no special permission for "hand tracking".  The open web will likely need a higher bar here - however, we should determine if any platforms or UAs would expect to treat non-simulated hand data in a special way before adding more API knobs.  For example, would we expect UAs to normalize hand sizes to reduce fingerprinting of users (no pun intended)?

Reply:
> If you chose "simulated" on a device with only a depth sensor to detect hands, would the UA be required to degrade the data in some way to below a level that would require a permissions prompt?

Not sure. I can imagine allowing UAs to make a choice here on when they show a prompt for simulated hands (but requiring a prompt for faithful hands)

> For example, would we expect UAs to normalize hand sizes to reduce fingerprinting of users (no pun intended)?

(we had fun with this pun at the face to face lol)

But yeah, for non-permissions cases I'd expect this.

Perhaps we should move this discussion to #7?